src/cli.ts

```diff
@@ -187,7 +195,7 @@ async function prepareSuites(inputs: string[]) {
       headless: options.headless,
       chromiumSandbox: options.sandbox,
       ignoreHTTPSErrors: options.ignoreHttpsErrors,
-    }
+    },
```

Comment:
DId we enable trailing comma in prettier?

Reply:
It seems to be the default (`es5`) in prettier 🤔 

https://prettier.io/docs/en/options.html#trailing-commas

Reply:
Hmmm, not sure how its picking it now.